### ISSUE TITLE
error TS2307 Cannot find module openapi3-ts

### ISSUE THREAD

**Issue by marioestradarosa** [strongloop/loopback-next#1695]
## Description / Steps to reproduce / Feature proposal
Scaffold a new app 
`lb4 app test`
`cd test`
`npm start`

See related comments
 https://github.com/strongloop/loopback-next/issues/1688#issuecomment-419688949
https://github.com/strongloop/loopback-next/issues/1688#issuecomment-419835882

## Current Behavior
After scaffolding a new app in lb4 version 0.22.x  and run `npm start` it complains with the following message:

````
node_modules/@loopback/rest/dist8/src/keys.d.ts:54:39 - error TS2307: Cannot find module
 '@loopback/openapi-v3-types/node_modules/openapi3-ts/dist/model/OpenApi'.

54     const API_SPEC: BindingKey<import("@loopback/openapi-v3-types/node_modules/
openapi3-ts/dist/model/OpenApi").OpenAPIObject>;
                 
````
## Expected Behavior
The app should start wihtout any problems.

## Work around
`cd ./node_modules/@loopback/openapi-v3-types`
`npm install`

**Comment by marioestradarosa** [comment]
@raymondfeng , the https://github.com/strongloop/loopback-next/pull/1689 didn't fix this issue. I have 0.22.2

**Comment by raymondfeng** [comment]
@marioestradarosa What version of `@loopback/rest` do you see? It should be `0.21.0`, which fixes the invalid import.

**Comment by marioestradarosa** [comment]
@raymondfeng  @loopback/rest: ^0.20.0

**Comment by marioestradarosa** [comment]
It is fixed now with @loopback/cli version: 0.22.3 . 

💯 @raymondfeng
